pkgs/development/python-modules/cfv/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
```

Comment:
It looks like the module is only used as application. In that case it should go in pkgs/by-name and use buildPythonApplication